OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -110,7 +110,8 @@ public override bool HandleMouseInput(MouseInput mi)
 							.WithHighestSelectionPriority();
 
 						var newSelection2 = SelectActorsInBox(World, worldRenderer.Viewport.TopLeft, worldRenderer.Viewport.BottomRight,
-							a => unit != null && a.Info.Name == unit.Info.Name && a.Owner == unit.Owner);
+							a => unit != null && World.RenderPlayer != null && unit.Owner == World.RenderPlayer
```

Comment:
`&& World.RenderPlayer != null` is not necessary.
If it is `null`, then `unit.Owner == World.RenderPlayer` would just be false.